linera-base/src/identifiers.rs

```diff
@@ -147,15 +156,53 @@ pub struct ChainId(pub CryptoHash);
 #[cfg_attr(with_testing, derive(Default))]
 pub struct BlobId(pub CryptoHash);
 
+impl BlobId {
+    /// Creates a new BlobId from a Blob
```

Comment:
```suggestion
    /// Creates a new `BlobId` from a `Blob`
```

Or "blob ID from a blob".

Or maybe: "Computes the blob's ID?"

We could also add to `Blob`'s and/or `BlobId`s documentation that it's content-addressed, i.e. the latter is the hash of the former.